pkg/controller/jobs/deployment/deployment_webhook.go

```diff
@@ -58,6 +62,8 @@ func (wh *Webhook) Default(ctx context.Context, obj runtime.Object) error {
 	log := ctrl.LoggerFrom(ctx).WithName("deployment-webhook")
 	log.V(5).Info("Applying defaults")
 
+	jobframework.ApplyDefaultLocalQueue(deployment.Object(), wh.queues.DefaultLocalQueue(deployment.ObjectMeta.Namespace))
```

Comment:
```suggestion
	ApplyDefaultLocalQueue(job.Object(), w.Queues.DefaultLocalQueue)
```
The same for other webhooks.